Build problem: Cannot find module '@oclif/plugin-help/lib/command'

## I'm submitting a...
<!-- Put an `x` in all the boxes that apply -->
- [x] Bug report
- [ ] Feature request
- [ ] Documentation issue or request
- [ ] Other... Please describe:

## Expected Behavior

After installing jovo-cli, and aws ask-cli, impossible to build, a command module seems to be missing.
Installing the module separately does not help either.

I tried to install  @oclif/plugin-help, @oclif/core, @oclif/command and @oclif/config, but no luck.

`jovo build`

## Current Behavior

```
$ jovo build
    Error: Unable to load configured help class "./dist/help/RootHelp.js", failed with message:
    Cannot find module '@oclif/plugin-help/lib/command'
    Require stack:
    - /usr/local/lib/node_modules/@jovotech/cli/dist/help/CommandHelp.js
    - /usr/local/lib/node_modules/@jovotech/cli/dist/help/RootHelp.js
    - /usr/local/lib/node_modules/@jovotech/cli/node_modules/@oclif/help/lib/util.js
    - /usr/local/lib/node_modules/@jovotech/cli/node_modules/@oclif/help/lib/command.js
    - /usr/local/lib/node_modules/@jovotech/cli/node_modules/@oclif/help/lib/index.js
    - /usr/local/lib/node_modules/@jovotech/cli/node_modules/@oclif/command/lib/command.js
    - /usr/local/lib/node_modules/@jovotech/cli/node_modules/@oclif/command/lib/index.js
    - /usr/local/lib/node_modules/@jovotech/cli/dist/index.js
    - /usr/local/lib/node_modules/@jovotech/cli/bin/run
```

## Your Environment

```
@jovotech/cli: 4.0.2 

Jovo packages of the current project :
  - @jovotech/cli-command-build: 4.0.1
  - @jovotech/cli-command-deploy: 4.0.1
  - @jovotech/cli-command-get: 4.0.1
  - @jovotech/cli-command-new: 4.0.1
  - @jovotech/cli-command-run: 4.0.1
  - @jovotech/cli-core: 4.0.1
  - @jovotech/common: 4.0.2
  - @jovotech/db-filedb: 4.0.1
  - @jovotech/filebuilder: 0.0.1
  - @jovotech/framework: 4.0.1
  - @jovotech/model: 4.0.0
  - @jovotech/model-nlpjs: 4.0.0
  - @jovotech/nlu-nlpjs: 4.0.1
  - @jovotech/output: 4.0.1
  - @jovotech/platform-core: 4.0.1
  - @jovotech/plugin-debugger: 4.0.1
  - @jovotech/server-express: 4.0.1
                                                                                
Environment:
  System:
    OS: macOS 11.6
  Binaries:
    Node: 15.13.0 - /usr/local/bin/node
    npm: 7.24.2 - /usr/local/bin/npm
```

--- a/cli/package.json
+++ b/cli/package.json
@@ -30,8 +30,8 @@
     "@jovotech/cli-core": "^4.0.1",
     "@oclif/command": "^1.8.0",
     "@oclif/config": "^1.17.0",
-    "@oclif/plugin-help": "^3.2.2",
-    "@oclif/plugin-not-found": "^1.2.4",
+    "@oclif/help": "^1.0.1",
+    "@oclif/plugin-not-found": "^2.2.3",
     "envinfo": "^7.8.1",
     "latest-version": "^5.1.0",
     "lodash.get": "^4.4.2"
@@ -63,7 +63,6 @@
   "oclif": {
     "bin": "jovo",
     "plugins": [
-      "@oclif/plugin-help",
       "@oclif/plugin-not-found"
     ],
     "helpClass": "./dist/help/RootHelp.js",

--- a/cli/src/help/CommandHelp.ts
+++ b/cli/src/help/CommandHelp.ts
@@ -1,9 +1,9 @@
 import { chalk, Log } from '@jovotech/cli-core';
 import { Command } from '@oclif/config';
-import BaseCommandHelp from '@oclif/plugin-help/lib/command';
-import { renderList } from '@oclif/plugin-help/lib/list';
+import BaseCommandHelp from '@oclif/help/lib/command';
+import { renderList } from '@oclif/help/lib/list'
 
-export default class CommandHelp extends BaseCommandHelp {
+export class CommandHelp extends BaseCommandHelp {
   /**
    * Mimicks the functionality of super.flags(), but customizes some minor
    * things such as output of flag parameters

--- a/cli/src/help/RootHelp.ts
+++ b/cli/src/help/RootHelp.ts
@@ -1,7 +1,7 @@
 import { chalk, isJovoCliError, JovoCliError, Log, PluginCommand } from '@jovotech/cli-core';
 import { Command, Topic } from '@oclif/config';
-import { HelpBase } from '@oclif/plugin-help';
-import CommandHelp from './CommandHelp';
+import { HelpBase } from '@oclif/help';
+import { CommandHelp } from './CommandHelp';
 
 export default class HelpOutput extends HelpBase {
   globalFlags: Record<string, Command.Flag> = PluginCommand.flags as Record<string, Command.Flag>;

